### DISCUSSION TITLE
Are the BEAMnrc examples (EX10MeVe and EX16MVp) Varian or Elekta accelerators?

### DISCUSSION THREAD

**Discussion by caioftportela** [nrc-cnrc/EGSnrc#775] [Questions] [answered]
Blackewalter and ftessier, good afternoon.

I hope that you be good.

I need a information about the BEAMnrc_Examples.

The EX10MeVe and EX16MVp is about Elekta or Varian accelerator?

Thanks a lot.

**Comment by rtownson** [comment, marked as answer]
These are just examples and do not include specifications of Elekta or Varian accelerators, which are proprietary. You will need to obtain those specifications from the manufacturer and then design the input file yourself. Ask for their Monte Carlo package.

**Reply by ftessier** [reply]
Indeed, these are generic examples. From the original report **PIRS-0509(H)** and **PIRS-0509(I)** (which are outdated and not included in the release anymore):

> The accelerators modelled are not real and not optimized except for teaching purposes.
We are trying to show what the code can do, and how it is done.
